src/pybind/mgr/cephadm/module.py

```diff
@@ -1560,7 +1560,37 @@ def _remove_daemon(self, name, host) -> str:
         self.cache.invalidate_host_daemons(host)
         return "Removed {} from host '{}'".format(name, host)
 
-    def _apply_service(self, spec) -> bool:
+    def _create_fn(self, service_type: str) -> Callable[..., str]:
+        try:
+            d: Dict[str, function] = {
+                'mon': self.mon_service.create,
+                'mgr': self.mgr_service.create,
+                'osd': self.osd_service.create,
+                'mds': self.mds_service.create,
+                'rgw': self.rgw_service.create,
+                'rbd-mirror': self.rbd_mirror_service.create,
+                'nfs': self.nfs_service.create,
+                'grafana': self.grafana_service.create,
+                'alertmanager': self.alertmanager_service.create,
+                'prometheus': self.prometheus_service.create,
+                'node-exporter': self.node_exporter_service.create,
+                'crash': self.crash_service.create,
+                'iscsi': self.iscsi_servcie.create,
```

Comment:
would now be a good time to fix the misspelling of "iscsi_service" that seems to have crept in via 20337c9b2a8b ?

Reply:
Nice spot! It will be fixed in https://github.com/ceph/ceph/pull/35068